channeldb/payments.go

```diff
@@ -1157,6 +1157,10 @@ func serializeHop(w io.Writer, h *route.Hop) error {
 		))
 	}
 
+	if h.AMP != nil {
```

Comment:
Code looks good, thanks! Can you update the commit message according to our [guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#ideal-git-commit-structure) please? Mainly adding the `channeldb:` prefix.